### PR TITLE
feat(navigation): Update navigation

### PR DESCRIPTION
**Pull request by jophish**
### Description

For [RET-1148](https://linear.app/valora/issue/RET-1148/[shell]-update-navigation-homewalletactivity). Updates navigation and home tabs to match designs. Also removes the Valora gradient from certain underlines throughout the app.

### Test plan

Unit and manual tested. See video below.

https://github.com/user-attachments/assets/c9c3e735-665c-4454-abe9-5ce0e4e06500

### Related issues

Fixes [RET-1148](https://linear.app/valora/issue/RET-1148/[shell]-update-navigation-homewalletactivity).

### Network scalability

If a new NetworkId and/or Network are added in the future, the changes in this PR will:

- [x] Continue to work without code changes, OR trigger a compilation error (guaranteeing we find it when a new network is added)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -350,7 +350,7 @@
       "emulator": {
         "type": "android.emulator",
         "device": {
-          "avdName": "Pixel_API_29_AOSP_x86_64"
+          "avdName": "Pixel_XL_API_29_NEW"
```
Comment (kathaypacific) [inline package.json]: is this change intentional?

File: src/components/TokenBalance.tsx [hunk 2]
```diff
@@ -201,16 +201,16 @@ export function AssetsTokenBalance({ showInfo }: { showInfo: boolean }) {
 
   return (
     <TouchableWithoutFeedback onPress={handleDismissInfo}>
-      <View testID="AssetsTokenBalance">
+      <View testID="AssetsTokenBalance" style={styles.row}>
         <View style={styles.row}>
-          <Text style={styles.walletTabTitle}>{t('bottomTabsNavigator.wallet.title')}</Text>
           {showInfo && (
```
Comment (kathaypacific) [inline src/components/TokenBalance.tsx]: are we keeping this info bubble? it looks kind of out of place without the title text... it's also not in the designs, so just wanted to check

File: src/earn/EarnHome.test.tsx [hunk 3]
```diff
@@ -196,7 +196,7 @@ describe('EarnHome', () => {
     ).toBeTruthy()
 
     fireEvent.press(getByText('tokenBottomSheet.filters.tokens'))
-    fireEvent.press(getByTestId('USDCSymbol'))
+    fireEvent.press(getByTestId('BottomSheetUSDCSymbol'))
```
Comment (kathaypacific) [inline src/earn/EarnHome.test.tsx:199]: if you are adding the "BottomSheet" prefix because there can be another element outside the bottom sheet that has test id "USDCSymbol", another option is to do something like `fireEvent.press(within(getByTestId('BottomSheet')).getByTestId('USDCSymbol'))` to lock the scope of the search. but not sure if this was the intention, so ignore if you just wanted to update the test id for readbility

File: src/home/ActionsCarousel.tsx [hunk 4]
```diff
@@ -120,6 +121,9 @@ function ActionsCarousel() {
 }
 
 const styles = StyleSheet.create({
+  container: {
+    zIndex: -1,
```
Comment (kathaypacific) [inline src/home/ActionsCarousel.tsx]: why is this needed? is it for the info bubble on the balance? i always feel like zIndex is a hack and makes the styling unpredictable (causes some head scratching especially if you're working on a component that isn't in this file, but is affected by this style which you don't see in the file you're working on) - if it's for the info bubble, perhaps we can put this style in the info bubble to reduce the effect / isolate the hack?

File: src/home/TabActivity.test.tsx [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+  }
+
+  it('renders activity tab correctly and fires initial actions', async () => {
+    const { store } = renderScreen()
+
+    await act(() => {
+      jest.runOnlyPendingTimers()
+    })
+
+    expect(store.getActions().map((action) => action.type)).toEqual(
```
Comment (kathaypacific) [inline src/home/TabActivity.test.tsx]: nit: a more concise alternative to `runOnlyPendingTimers` is to wrap this `expect` in a `await waitFor()`

File: src/home/TabActivity.test.tsx [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+    ...originalModule,
+    __esModule: true,
+    default: {
+      ...originalModule.default,
+      defaultNetworkId: 'celo-alfajores',
+    },
+  }
+})
+
+const mockBalances = {
```
Comment (kathaypacific) [inline src/home/TabActivity.test.tsx]: do we need this mock if we're not testing anything related to tokens?

File: src/home/TabActivity.tsx [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+function TabActivity(_props: Props) {
+  const appState = useSelector(appStateSelector)
+  const isLoading = useSelector((state) => state.home.loading)
+
+  const insets = useSafeAreaInsets()
+
+  const dispatch = useDispatch()
+
+  useEffect(() => {
+    dispatch(visitHome())
```
Comment (kathaypacific) [inline src/home/TabActivity.tsx]: this reads weirdly 🙈 do we still need it? maybe it can be on the home screen still?

File: src/home/TabActivity.tsx [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+
+  const keyExtractor = (_item: any, index: number) => {
+    return index.toString()
+  }
+
+  const refresh: React.ReactElement<RefreshControlProps> = (
+    <RefreshControl refreshing={isLoading} onRefresh={onRefresh} colors={[colors.primary]} />
+  ) as React.ReactElement<RefreshControlProps>
+
+  const notificationBoxSection = {
```
Comment (kathaypacific) [inline src/home/TabActivity.tsx]: this probably isn't needed anymore?

File: src/home/TabActivity.tsx [hunk 9]
```diff
@@ -0,0 +1,103 @@
...
+        testID="WalletHome/SectionList"
+      />
+    </SafeAreaView>
+  )
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    position: 'relative',
```
Comment (kathaypacific) [inline src/home/TabActivity.tsx]: i think this is the default value :)

File: src/home/TabHome.tsx [hunk 10]
```diff
@@ -69,13 +44,6 @@ function TabHome(_props: Props) {
...
   }
 
-  // Scroll Aware Header
-  const scrollPosition = useSharedValue(0)
-
-  const handleScroll = useAnimatedScrollHandler((event) => {
-    scrollPosition.value = event.contentOffset.y
-  })
-
   const tryImportContacts = async () => {
```
Comment (kathaypacific) [inline src/home/TabHome.tsx:47]: should we also maybe remove this contact mapping stuff?

File: src/navigator/Headers.tsx [hunk 11]
```diff
@@ -228,9 +226,7 @@ export const tabHeader: NativeStackNavigationOptions = {
   headerRight: () => {
     return (
       <View style={[styles.topElementsContainer, { marginRight: Spacing.Tiny4 }]}>
-        <QrScanButton testID="WalletHome/QRScanButton" />
```
Comment (kathaypacific) [inline src/navigator/Headers.tsx:231]: maybe an existential question but do we expect MS apps not to need a QR scanner? i guess it makes sense if we expect users to use only the embedded dapp

**Review by kathaypacific — APPROVED** [review]
LGTM! 🚀 just left some small nits

File: src/home/TabHome.tsx (hunk 10 above)
Comment (jophish) [reply src/home/TabHome.tsx:47]: I was gonna keep it in merely to reduce diffs, and in case we ever want to include it later, since it's basically just a no-op atm.

File: package.json (hunk 1 above)
Comment (jophish) [reply package.json]: oooops, no! this is what i use for e2e tests, forgot to change it back this time :)

File: src/components/TokenBalance.tsx (hunk 2 above)
Comment (jophish) [reply src/components/TokenBalance.tsx]: yeah good question -- checked with kayla, and i think we can get rid of this pending an OK from laura,

File: src/earn/EarnHome.test.tsx (hunk 3 above)
Comment (jophish) [reply src/earn/EarnHome.test.tsx:199]: This was specifically for e2e tests, which I struggled with for a while. Since we moved the quick actions to the wallet screen, there would be two elements with the `cUSDSymbol` test ID when opening the "add" quick action. Strangely, when both the bottom sheet and the wallet balances were showing, the tests were unable to find even a single element with the given test ID. Using the helper fn we have to grab elements at a particular index (for presumably this very use case) also failed to find any elements..

File: src/home/ActionsCarousel.tsx (hunk 4 above)
Comment (jophish) [reply src/home/ActionsCarousel.tsx]: This was for the info bubble next to the wallet balance for dapp positions; it would appear underneath the quick actions, and only changing the zIndex of the info bubble (but not the quick actions) appeared to have no effect. fortunately since we're getting rid of the info bubble, we can remove this hack!

File: src/home/TabActivity.tsx (hunk 7 above)
Comment (jophish) [reply src/home/TabActivity.tsx]: good catch, yeah we can remove this

File: src/navigator/Headers.tsx (hunk 11 above)
Comment (jophish) [reply src/navigator/Headers.tsx:231]: yeah i had the same thought... i think it's in fact still accessible through the "receive" quick action, if enabled.
